git/foundations_git/git_basics.md

```diff
@@ -200,4 +200,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
--   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+-   [Make Atomic Git Commits] (https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/)
```

Comment:
Could we add a short description of sorts here? For newer users, they may not know what "atomic commits" are or why it'd be worth reading this additional resource.

Reply:
This article titled _Make Atomic Git Commits_ by Aleksandr Hovhannisyan provides further information on what Atomic Commits are, why they are considered to be Git best practices, and how they can help you both improve the quality of your work and make life easier for you and other developers on your team.

Reply:
What if we updated this resource to something like:

```
- [Make Atomic Git Commits] (https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) goes over what exactly atomic commits are and how they can be beneficial when working on a team.
```

Additionally, after going through some of the lessons we have on Git, I feel like this resource might be better in one of the Intermediate Git lessons in our JavaScript course. This resource mentions a good amount of stuff that gets covered in those lessons.

Reply:
The description you suggest seems fine to me. I am not that further into the TOP content, so I can't comment on placing this resource in the JavaScript course. However, you know the contents better than I do, so go ahead. I'm just glad the article can benefit someone down the line.

Reply:
Hey there, just checking on this pending PR. Just wondering what's the next thing to do. Are you going to take care of it or should I proceed with something else? I'm sorry in advance if this is bothersome, I'm just a noob.  I'll be looking forward to your reply. Thank you.

Reply:
Hey, not a bother! Sometimes it can just take a bit to get back to things.

So after looking at some of our other git lessons, this could probably fit in our Commit Messages lesson in Foundations. Would you want to make an update to this PR to move the additional resource to that lesson?

Reply:
Hey there, sure! It sounds good. I am not 100% sure how to proceed, though. Should I submit another PR from the Commit Messages Lesson and discard this one?

Reply:
Assuming you made this PR through GitHub rather than a local clone that you pushed to your fork, you could:

1. revert the git basics file in your `patch-1` branch on your fork (edit it, then select "Commit directly to patch-1 branch")
2. Update the commit messages file, and commit directly to patch-1 again.

If you want to do all of that in a single commit via GitHub, if you go to the `patch-1` branch of your fork and press the period key `.` it'll open your fork in the GitHub editor (it'll basically look similar to your VS Code experience). May be slightly less straight forward, but it's an option.

If you need any other help/questions just let me know!